src/schemata/definitions/field.yml

```
@@ -15,13 +15,18 @@ allOf:
       ui:options:
         type: object
         description: |
-          Display configuration options for certain field types and widget settings. If a field or widget does not support `ui:options`, the setting is ignored.
+          Configuration options for certain field types and widget settings. If a field or widget does not support `ui:options`, the setting is ignored.
         properties:
           label:
             type: boolean
             default: true
             description: |
               A flag indicating whether a field of type `boolean` has a display label. Setting this to `false` will display either only a checkbox or switch control element, without any visible label.
+        customOptions:
+          type: boolean
+          default: false
+          description: |
+            A flag indicating whether fields of type `string` or `number` that are configured with `enum` options can be updated with a value that is not among the specified options. If set to `true`, the user can both select from the specified options, as well as enter any custom value that satisfies the field schema. Fields that are not of type `string` or `number` ignore this setting.
```

lets do this, and flip the default to `true`
```suggestion
          strictEnum:
            type: boolean
            default: true
            description: |
              A flag indicating whether fields of type `string` or `number` that are configured with `enum` options can be updated with a value that is not among the specified options. If set to `true`, the user can both select from the specified options, as well as enter any custom value that satisfies the field schema. Fields that are not of type `string` or `number` ignore this setting.
```